packages/manager/apps/telecom/src/app/telecom/pack/xdsl/meetings/xdsl-meetings.html

```diff
@@ -0,0 +1,67 @@
+<div class="m-4">
+    <header class="widget-presentation-header">
+        <h2
+            class="widget-presentation-title"
+            data-translate="xdsl_meeting_slots"
+        ></h2>
+    </header>
+
+    <!-- LOADER -->
+    <div class="text-center" data-ng-if="$ctrl.loading">
+        <oui-spinner></oui-spinner>
+    </div>
+
+    <oui-message type="error" data-ng-if="$ctrl.errorMessage">
+        <span data-ng-bind-html="$ctrl.errorMessage"></span>
+    </oui-message>
+
+    <oui-message type="success" data-ng-if="$ctrl.successMessage">
+        <span data-ng-bind-html="$ctrl.successMessage"></span>
+    </oui-message>
+
+    <div
+        class="widget-presentation py-3 move-number-config ft-meeting"
+        data-ng-if="$ctrl.showMeetingSlots && !ctrl.loading && !$ctrl.successMessage"
+    >
+        <strong data-translate="xdsl_meeting_description"></strong>
+        <div
+            class="mt-3"
+            data-ng-show="!$ctrl.select.meetingSlots.slot"
+            data-translate="xdsl_meeting_programmed_meeting_select"
+        ></div>
+        <div
+            class="mt-3"
+            data-ng-show="$ctrl.select.meetingSlots.slot && !$ctrl.select.meetingSlots.fakeMeeting"
+            data-ng-bind-html="$ctrl.meetingSelectMessage"
+        ></div>
+        <div class="row">
+            <div
+                class="oui-box oui-box_light m-3 col-md-2"
+                data-ng-repeat="meeting in $ctrl.meetings track by $index"
+                style="height: 15.625rem;"
```

Comment:
We prefer to avoid style property.

Reply:
Yes I understand, but I need to have the same size for all boxes which is not the case because all boxes have not the same number of available meetings

Reply:
`style="height: 15.625rem;"` is not dynamic and don't depends on number of numbers ? I don't get it.

Reply:
It's the idea.